.tx/config

```diff
@@ -1,6 +1,13 @@
 [main]
 host = https://www.transifex.com
 
+[suomifi-design-system.alert-json]
+file_filter = locale/<lang>/alert.json
+minimum_perc = 0
+source_file = locale/af/alert.json
+source_lang = af
+type = KEYVALUEJSON
+
```

Comment:
We have added new items at the end of the list, instead of the beginning.

Same for the **SingleSelect**, but that can be fixed in separate PR as there also other fixes/changes needed for it.